notebooks/official/model_monitoring/model_monitoring.ipynb

```diff
@@ -36,7 +36,7 @@
         "<table align=\"left\">\n",
```

Comment:
<p>Don't you need to undeploy the model before you delete the endpoint?</p><p> </p><p>What about deleting the BQ table with the monitoring data?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/841/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>

Reply:
<p>I think deleting the endpoint via gcloud takes care of undeploying the model but I'm in the process of verifying that now. Also, taking a look at how to clean up the BQ table, good catch!</p><p> </p><p>Would it be possible to approve this PR to unblock it and I'll submit a follow up, smaller PR tomorrow to fix the clean up cell?</p>